kernel/prepare/pkg.yaml

```diff
@@ -9,44 +9,26 @@ steps:
         destination: linux.tar.xz
         sha256: "{{ .linux_sha256 }}"
         sha512: "{{ .linux_sha512 }}"
+      - url: https://github.com/a13xp0p0v/kernel-hardening-checker/archive/{{ .kspp_ref }}.tar.gz
+        destination: kspp.tar.gz
+        sha256: "{{ .kspp_sha256 }}"
+        sha512: "{{ .kspp_sha512 }}"
     env:
       ARCH: {{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}x86_64{{ else }}unsupported{{ end }}
     prepare:
       - |
         tar -xJf linux.tar.xz --strip-components=1
         rm linux.tar.xz
 
-        mkdir /bin
-        ln -sv /toolchain/bin/bash /bin/bash
-        ln -sv /toolchain/bin/bash /bin/sh
-        ln -sv /toolchain/lib /lib
-        mkdir -p /usr/bin \
-            && ln -sf /toolchain/bin/env /usr/bin/env \
-            && ln -sf /toolchain/bin/true /bin/true \
-            && ln -sf /toolchain/bin/false /bin/false \
-            && ln -sf /toolchain/bin/pwd /bin/pwd
-
-        # Ensure that `make menuconfig` works.
-        ln -s /toolchain/bin/awk /usr/bin/awk
-
-        mkdir -p /usr/lib/pkgconfig
-        ln -s /toolchain/include /usr/include
-
         make mrproper
       - |
-        cd /toolchain && git clone https://github.com/a13xp0p0v/kernel-hardening-checker.git
+        mkdir -p /toolchain/kernel-hardening-checker
+        tar -xzf kspp.tar.gz --strip-components=1 -C /toolchain/kernel-hardening-checker
     install:
       - |
         mkdir -p /src
         cp -a . /src/
+        rm -r /tmp/build/*
```

Comment:
why are we doing this? We shouldn't rely on bldr internal hacks

Reply:
Hmm, the original copied most directories but not /tmp. Maybe only wanted directories (and not bin lib etc.) should be added to the /rootfs, and then it'll be layered on top of base when used in kernel/build for example?